dll/win32/iphlpapi/icmp.c

```diff
@@ -905,6 +983,10 @@ DWORD WINAPI IcmpSendEcho(
 
             /* Prepare for the next packet */
             ier++;
+#ifndef __REACTOS__
+            ip_header=(struct ip*)(((char*)ip_header)+sizeof(ICMP_ECHO_REPLY));
```

Comment:
This code is incorrect?
We don't need it?

Reply:
I don't know: out of scope of this PR, see [r72759](https://git.reactos.org/?p=reactos.git;a=commit;h=51d329b7e13d39e32f4bcc3f795b7c7e35af0d7a).